### PR TITLE
Oracle improvements

### PR DESCRIPTION
**Pull request by kattylucy**
Add improvements to oracle setup

- Change Pricing tab by default
- Oracle feeders => Oracle providers
- Adapt configuration change component from Pool managers setup for Oracle feeders:
Header: Oracle update threshold
Subtitle: `Determine how many oracle providers are required before a pricing update is finalized and will become reflected in the NAV.
- Toast Set oracle prices => Set oracle providers
- Write-off policy
 - Move Write-off policy section from Configuration tab to Pricing tab (below oracle providers)
 - Remove At least one write-off activity is required subtitle

#2242 

- [ ] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### CODE REVIEW

**Comment by kattylucy** [comment]
Screenshot 2024-07-02 at 1 40 55 PM

**Comment by hieronx** [comment]
@kattylucy good question, in this case 1 should be allowed!

**Comment by hieronx** [comment]
Two more additioanl details:

1. Can you move the threshold below the table of oracle providers (again matching the Access tab)?
2. Above the table, can you add the following line, in the same style as the selected text in the screenshot below: "Add or remove addresses that can provide oracle updates for the onchain NAV."
Screenshot 2024-07-02 at 21 04 39

**Comment by kattylucy** [comment]
I made all the changes but Im having issues creating or updating pools (this is on demo). Any idea? what can be the problem? - having the same issue on main branch 

@hieronx 

Screenshot 2024-07-03 at 9 26 28 AM

**Comment by hieronx** [comment]
@kattylucy try updating your extension/wallet in the browser

For the implementation
- "Add or remove addresses that can provide oracle updates for the onchain NAV." should be above the table, not below it.
- The copy of the Minimum feeders section is off, it should be

Header: Oracle update threshold
Subtitle: `Determine how many oracle providers are required before a pricing update is finalized and will become reflected in the NAV.

**Comment by kattylucy** [comment]
Screenshot 2024-07-03 at 1 40 59 PM

**Comment by hieronx** [comment]
It looks great! Just forgot, can you change `out of x managers` to `out of x providers`?

**Review by hieronx — COMMENTED** [review]
@onnovisser should review code but from product perspective looks great to me now!

**Comment by kattylucy** [comment]
Sounds good. I will squash the commits so is easier to review - TY @hieronx

File: centrifuge-app/src/pages/IssuerPool/Access/ChangeTreshold.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  fieldName,
+  signersFieldName,
+  minThreshold = 2,
+  validate,
+  disabled,
+  type,
+}: ChangeThresholdProps) => {
+  const form = useFormikContext<any>()
+  const threshold = getNestedValue(form.values, fieldName)
+  const signers = getNestedValue(form.values, signersFieldName)
```
Comment (onnovisser) [inline centrifuge-app/src/pages/IssuerPool/Access/ChangeTreshold.tsx:34]: formik exports a `get` function for this

File: centrifuge-app/src/pages/IssuerPool/Access/ChangeTreshold.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  validate?: (value: any) => string | undefined
+  disabled?: boolean
+  type: string
+}
+
+const getNestedValue = (obj: any, path: string) => {
+  return path.split('.').reduce((acc, part) => acc && acc[part], obj)
+}
+
+const ChangeThreshold = ({
```
Comment (onnovisser) [inline centrifuge-app/src/pages/IssuerPool/Access/ChangeTreshold.tsx:21]: let's do a named export instead

File: centrifuge-app/src/pages/IssuerPool/Access/ChangeTreshold.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      <Text as="p" variant="body2" color="textSecondary">
+        {secondaryText}
+      </Text>
+
+      <Shelf gap={2}>
+        {isEditing && (
+          <Box maxWidth={150}>
+            <Field
+              name={fieldName}
+              validate={validate ?? min(minThreshold, `Multisig needs at least ${minThreshold} signers`)}
```
Comment (onnovisser) [inline centrifuge-app/src/pages/IssuerPool/Access/ChangeTreshold.tsx:49]: could change the default error message to `needs at least ${minThreshold} ${type}`

**Comment by onnovisser** [comment]
looking good, just a couple nits

**Review by onnovisser — APPROVED** [review]

**Pull request merged**